Assets/Python/Afforess/ANewDawnSettings.py

```
@@ -1,334 +1,470 @@
-	changedTerrainDamage(ANewDawnOpt, ANewDawnOpt.isTerrainDamage())
-	changedAllowTerraforming(ANewDawnOpt, ANewDawnOpt.isAllowTerraforming())
-	changedReforestation(ANewDawnOpt, ANewDawnOpt.isReforestation())
-	changedSeaTunnels(ANewDawnOpt, ANewDawnOpt.isSeaTunnels())
-	changedWarPrizes(ANewDawnOpt, ANewDawnOpt.isWarPrizes())
-	changedFlexibleDifficultyAI(ANewDawnOpt, ANewDawnOpt.isFlexibleDifficultyAI())
-	changedHistoricalAccurateCalendar(ANewDawnOpt, ANewDawnOpt.isHistoricalAccurateCalendar())
-	changedInfrastructureIgnoresImprovements(ANewDawnOpt, ANewDawnOpt.isInfrastructureIgnoresImprovements())
-
-	ANewDawnOpt.setPlayerColor(0)
-	ANewDawnOpt.setCurrentDifficulty(0)
+    print
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)